README.md

```
@@ -139,8 +139,13 @@ The appearance of the popup is shown in the images below:
 
 ## ⚠️ Disclaimer
 
-**This software was created to automate participation in mini-games on the channel — not to exploit or abuse the system.**
+This extension was built to make participating in mini-games more convenient for individual users.  
+It’s not designed for abuse, large-scale automation, or use with alt accounts.  
+
+> The channel owner has stated that scripts are not allowed. Use at your own risk!
+
```

markdownlint-cli2 (0.17.2)

Blank line inside blockquote

(MD028, no-blanks-blockquote)